tests_e2e/test_e2e.py

```diff
@@ -28,10 +28,12 @@ def deleteBoard():
 def app_with_temp_board():
     # Load dotenv file before being used/referenced
     file_path = find_dotenv('.env')
+    os.environ['LOGIN_DISABLED'] = 'True'
```

Comment:
You can get rid of this one since you have the same thing on line 36 (correctly placed **after** loading the .env file